### ISSUE TITLE
🐛 Bug Report: Don't Show API Responses for Errors

### ISSUE THREAD

**Issue by ItzNotABug** [appwrite/console#974]
### 👟 Reproduction steps

1. Delete an organization [assume org. id as `xyz`]
2. Go to the Console via direct url i.e. https://cloud.appwrite.io/console/project-xyz/
    OR simply use the above link with a wrong project id
4. Error is displayed as below -

<h2>Invalid Argument</h2>
<h3>Project with the requested ID could not be found. Please check the value of the X-Appwrite-Project header to ensure the correct project ID is being used.</h3>

---
Console should instead show a simple - 
**Project does not exist or was recently deleted or you might have been removed from the organization.**

### 👍 Expected behavior

Show a more simpler, understandable error message.

### 👎 Actual Behavior

API response is shown directly as Error which can be confusing to some users.

### 🎲 Appwrite version

Appwrite Cloud

### 💻 Operating system

Linux

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by naman1608** [comment]
@ItzNotABug agreed, the error message to be displayed should be different from the API error response. Can I try this?

**Comment by ItzNotABug** [comment]
Hey @naman1608,
I think one of a core member needs to bump in here since there are multiple cases where this issue can happen around the console.

A direct/hardcoded message based on the returned error won't be enough.

**Comment by eldadfux** [comment]
Starting today this repo is archived. Please raise any issues with the new Appwrite console on the main platform repo: https://github.com/appwrite/appwrite
